code/modules/mob/living/carbon/carbon.dm

```diff
@@ -215,8 +210,8 @@
 			AdjustKnockdown(-3)
 			playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 			M.visible_message( \
-				"<span class='notice'>[M] shakes [src] trying to wake [t_him] up!</span>", \
-				"<span class='notice'>You shake [src] trying to wake [t_him] up!</span>", \
+				"<span class='notice'>[M] shakes [src] trying to wake \him up!</span>", \
```

Comment:
Well since it's tested and working as shown above I don't see the problem
It'll change twice if I made it a separate PR for ATOMICITY's sake too